ompi/mpi/man/man3/MPI_Dims_create.md

```diff
@@ -0,0 +1,93 @@
+# Name
+
+`MPI_Dims_create` - Creates a division of processors in a Cartesian
+grid.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Dims_create(int nnodes, int ndims, int dims[])
+```
+
+## Fortran Syntax
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_DIMS_CREATE(NNODES, NDIMS, DIMS, IERROR)
+    INTEGER    NNODES, NDIMS, DIMS(*), IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_Dims_create(nnodes, ndims, dims, ierror)
+    INTEGER, INTENT(IN) :: nnodes, ndims
+    INTEGER, INTENT(INOUT) :: dims(ndims)
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+
+# Input Parameters
+
+* `nnodes` : Number of nodes in a grid (integer).
+* `ndims` : Number of Cartesian dimensions (integer).
+
+# In/Out Parameter
+
+* `dims` : Integer array of size ndims specifying the number of nodes in each
+dimension.
+
+# Output Parameter
+
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+For Cartesian topologies, the function `MPI_Dims_create` helps the user
+select a balanced distribution of processes per coordinate direction,
+depending on the number of processes in the group to be balanced and
+optional constraints that can be specified by the user. One use is to
+partition all the processes (the size of `MPI_COMM_WORLD's` group) into
+an n-dimensional topology.
+The entries in the array `dims` are set to describe a Cartesian grid
```

Comment:
I think that these are supposed to be discrete paragraphs in the Description section.